core/domain/exp_jobs_one_off_test.py

```diff
@@ -414,6 +414,186 @@ def test_nonhuman_committers_not_counted(self):
+        """Test that if only non-revert commits are made by
+        contributor then the contributions summary shows same
+        exact number of commits for that contributor's ID
+        """
+
+        self.signup(self.EMAIL_B, self.USERNAME_B)
+        user_a_id = self.get_user_id_from_email(self.EMAIL_B)
+
+        # Let USER A make three commits.
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, user_a_id, title='Exploration Title')
+
+        exp_services.update_exploration(
+            user_a_id, self.EXP_ID, [{
+                'cmd': 'edit_exploration_property',
+                'property_name': 'title',
+                'new_value': 'New Exploration Title'
+            }], 'Changed title.')
+
+        exp_services.update_exploration(
+            user_a_id, self.EXP_ID, [{
+                'cmd': 'edit_exploration_property',
+                'property_name': 'objective',
+                'new_value': 'New Objective'
+            }], 'Changed Objective.')
+
+        # Run the job to compute contributors summary
+        job_id = exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob\
+            .create_new()
+        exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        exploration_summary = exp_services.get_exploration_summary_by_id(
+            exploration.id)
+
+        self.assertEqual(
+            3, exploration_summary.contributors_summary[user_a_id])
+
+    def test_contributors_with_only_reverts_not_included(self):
+        """Test that if only reverts are made by contributor then the
+        contributions summary shouldn’t contain that contributor’s ID
+        """
+
+        # Sign up two users.
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+        self.signup(self.EMAIL_B, self.USERNAME_B)
+        user_b_id = self.get_user_id_from_email(self.EMAIL_B)
+
+        # Let USER A make three commits.
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, user_a_id, title='Exploration Title 1')
+
+        exp_services.update_exploration(
+            user_a_id, self.EXP_ID, [{
+                'cmd': 'edit_exploration_property',
+                'property_name': 'title',
+                'new_value': 'New Exploration Title'
+            }], 'Changed title.')
+        exp_services.update_exploration(
+            user_a_id, self.EXP_ID, [{
+                'cmd': 'edit_exploration_property',
+                'property_name': 'objective',
+                'new_value': 'New Objective'
+            }], 'Changed Objective.')
+
+        # Let the second user revert version 3 to version 2
+        exp_services.revert_exploration(user_b_id, self.EXP_ID, 3, 2)
+
+        # Run the job to compute the contributors summary.
+        job_id = exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob\
```

Comment:
we don't use '\' to split into multi lines. So use "pylint: disable=line-too-long" for this line.